examples/kubernetes/README.md

```diff
@@ -1,170 +1,215 @@
-# Kubernetes Network Policy + Cilium
+# Kubernetes Network Policy + (L4) Ingress Controller with Cilium
 
 This tutorial will show you how you can have full IPv6 connectivity between
```

Comment:
This tutorial demonstrates how to set up Kubernetes using Cilium to:
  - Provide IPv4 & IPv6 network connectivity
  - Enforce container/pod labels based network security policies (using the v1beta "Network Policy" object)
  - Run the BPF based kube-proxy replacement for services and ingress objects